Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -247,23 +247,64 @@ public string GetLabelText()
         }
 
         public async Task LoadAsync()
-        {   
+        {
+            // Don't do anything if the config is already loaded.
+            if (config != null)
+                return;
+
             config = await EpicOnlineServices.Config.GetAsync<T>();
         }
 
+        public void Load()
+        {
+            Task.Run(LoadAsync).GetAwaiter().GetResult();
```

Comment:
Noting to self; we should research and apply the Task Factory so we can pass a Synchronization context. I'll create a task to adapt that, feel free to resolve this conversation.

Reply:
Made task #EOS-2083